plugin/evm/validators_default_test.go

```diff
@@ -0,0 +1,41 @@
+package evm
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/flare-foundation/coreth/params"
+)
+
+func TestGetDefaultValidators(t *testing.T) {
+	tests := []struct {
+		chainID            *big.Int
+		weightOfAValidator uint64
+		numberOfValidators int
+	}{
+		{
+			chainID:            params.CostonChainID,
+			weightOfAValidator: costonValidatorWeight,
+			numberOfValidators: 5,
+		},
+		{
+			chainID:            params.SongbirdChainID,
+			weightOfAValidator: songbirdValidatorWeight,
+			numberOfValidators: 20,
+		},
```

Comment:
We should also have a test with the FlareChainID and one with a custom validator set. For the test with the custom validator set, since it involves an environment variable, it could make sense to do it outside of the table-driven-test here, and instead write this test in a separate test function.

Reply:
Ok, but for FlareChainID an error is expected as the list is nil right now. Should I write a separate one for that as well or have checks inside the loop itself to do `require.Error()`  ?

Reply:
Yep, you can add an attribute called `wantErr` of type `require.ErrorAssertionFunc` and set it to `require.NoError` for other tests, and `require.Error` for this one :)